### PR TITLE
Enable ROS 2 CI and lint Github actions

### PR DESCRIPTION
**Pull request by thomas-moulard**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by thomas-moulard** [comment]
AFAIK, we need to merge the first PR to start having pipelines. My fork with this commit is gren:
https://github.com/thomas-moulard/launch_ros_sandbox/pull/1/checks?check_run_id=266322178

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Test launch_ros_sandbox
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    # Run every hour
```
Comment (prajakta-gokhale) [inline .github/workflows/test.yml]: Do we need to run these frequently?

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  ament_lint:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+          linter: [copyright, flake8, pep257]
+    steps:
+    - uses: actions/checkout@v1
+    - uses: thomas-moulard/setup-ros2@0.0.3
```
Comment (prajakta-gokhale) [inline .github/workflows/lint.yml]: Should these repos be moved to a shared team-wide org?

File: .github/workflows/test.yml (hunk 1 above)
Comment (thomas-moulard) [reply .github/workflows/test.yml]: Yes, I'd like to get an autobuild every hour to detect dependencies breakage.

**Comment by thomas-moulard** [comment]
Apparently, I triggered a bug in GH action, if I push a PR from the same repo, pipelines get kicked off. Let's continue on the other PR.

**Pull request closed without merging**